devtools/maven/src/main/java/io/quarkus/maven/NativeImageMojo.java

```diff
@@ -333,7 +337,10 @@ public void accept(ConfigBuilder configBuilder) {
                     configs.add("quarkus.native.debug-symbols", debugSymbols.toString());
                 }
                 if (disableReports != null) {
-                    configs.add("quarkus.native.disable-reports", disableReports.toString());
+                    configs.add("quarkus.native.enable-reports", new Boolean(!disableReports).toString());
```

Comment:
Is it safe? Could `disableReports` be null?

Reply:
`disableReports` null on line 340? there is a null check on line 339

Reply:
Oops sorry :).